Splitter: how to implement custom scrolling (#6265)
DevExpress/devextreme-documentation: concepts/05 UI Components/Splitter/20 Implement Custom Scrolling/00 Implement Custom Scrolling.md

> Splitter displays native scroll bars when a pane content is smaller than the pane itself (for instance, as a result of resize).

Reviewer: I guess it should be the other way - pane content should be **bigger** than the pane. Then you'll see scrollbars. 

```suggestion
Splitter displays native scroll bars when pane content doesn't fit into the pane. This often happens as a result of a resize operation.
```

> Splitter displays native scroll bars when pane content doesn't fit into the pane. This often happens as a result of a resize operation.

Reviewer 2: ```suggestion
Splitter displays native scroll bars when pane content doesn't fit into the pane, for example, when resizing a pane.
```

> You can replace native scroll bars with DevExtreme ScrollView component, which allows you to additionally configure scrolling. For example, you can specify [when to show a scroll bar](/Documentation/ApiReference/UI_Components/dxScrollView/Configuration/#showScrollbar). For more information about the available options, refer to the [ScrollView API section](/Documentation/ApiReference/UI_Components/dxScrollView/Configuration/).

Reviewer: I'm not sure we should talk about "native scroll bars" as equivalent to "ScrollView component" (as in "replace native scroll bars with ScrollView"). The component is a container that hosts some content and also displays scrollbars. So I wouldn't say that you can just replace one with the other. 

```suggestion
You can hide native scroll bars and use a DevExtreme ScrollView component. This component includes extended scroll bar settings. For example, you can use different [scroll bar visibility modes](/Documentation/ApiReference/UI_Components/dxScrollView/Configuration/#showScrollbar). For more information about the available options, refer to the [ScrollView API section](/Documentation/ApiReference/UI_Components/dxScrollView/Configuration/).
```

                                    const $scrollView = $("<div/>");
                                    $scrollView.append($("<div/>").html(htmlContent));
                                    $scrollView.dxScrollView({
                                        width: "100%",
                                        height: "100%",
                                        direction: "both",
                                    });
                                    return $scrollView;
                                },
                            },
                        ],
                    },
                },
                {
                    text: "Pane 4",
                },
            ],
        });
    });

    <!-- tab: index.css -->
    .caption {
        width: 330px;
        height: 330px;
        border: dashed;
        padding: 5px;
    }

    #container {
        height: 400px;
    }

>     .dx-item-content dx-splitter-item-content {

Reviewer 3: dot is missed. Check it in all approaches
